Storage/Storage/Model/ShipmentTrackingProviderGroup+CoreDataProperties.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ShipmentTrackingProviderGroup+CoreDataProperties.swift
+//  Storage
+//
+//  Created by Cesar Tardaguila on 14/3/2019.
+//  Copyright © 2019 Automattic. All rights reserved.
+//
```

Comment:
Style nit → remove the header comment here ☝️

Reply:
🤦‍♂️ Argh! Sorry about that